admin-docs/metadata-extraction.md

```diff
@@ -0,0 +1,101 @@
+
+### Prerequisites
+
+Before using the metadata extraction tool, there are a few prerequisites to consider. These include access to the tool, the type of data it works with, and your familiarity with the document content you want to extract.
+
+- **Access**: This tool is in the beta phase and isn’t activated automatically for all Uwazi instances. Contact us if you’d like to access it during this phase. Note that when activated, the tool is only accessible to users with the roles of admin or editor.
+
+- **Available data**: The tool currently works with PDF documents and can extract information to specific template properties. Supported properties include text, numeric, date, single select, multiple select, and relationship. Your instance should have appropriate templates with documents available for metadata extraction.
+
+- **Knowledge of document content**: It’s important to have a clear understanding of the kind of information you want to extract, such as specific properties or categories relevant to your documents. This preparation will help ensure a smooth and accurate extraction process.
+
+## How to use metadata extraction effectively
+
+This step-by-step guide will walk you through the process of extracting metadata from your documents. First, initiate the metadata extraction by selecting the relevant properties from the available templates. The tool will then generate and present the extracted metadata as suggestions. Finally, review the suggestions, accepting those that are correct and refining any that need adjustments. Following these steps will help ensure that the extracted metadata is both accurate and relevant.
+
+### Creating a metadata extractor
+
+Step 1: Go to **Settings** ![Settings icon](images/cog-solid.png) and select **Metadata Extraction**.
+
+Step 2: Click on **Create Extractor**.
+
+Step 3: In the dialog window that appears, enter a name for the extractor in the input field.
+
+Step 4: From the list of available templates and properties, choose the property to be used for metadata extraction, then click **Next**.
+
+Step 5: Review your selections and click **Create**.
+
+Once the system generates the extractor, it will appear in the table with its name, selected property, and associated templates.
+
+### Reviewing suggestions
+
+After creating a metadata extractor, it will be visible in the table listing all previously created extractors. The next step is to review the suggestions generated by the system. To do this, select **Review** in the corresponding table row to open the interface for reviewing suggestions from the selected metadata extractor.
+
+A table with the following columns will be displayed:
```

Comment:
I think it is important to explain how the "Find suggestions" button work. The blank state of the table is before they ran any training/extraction, so it won't look like described here.

I also think it is important to give a basic notion on how to properly use this feature:
1. Label some data, around 30-40 samples.
2. Run "Find suggestions"
3. Process some suggestions, particularly those that the ML model is not able to properly understand have more value than just confirming the patterns that the model is already doing well.
4. Back to step 2.

Reply:
> The blank state of the table is before they ran any training/extraction, so it won't look like described here.

@txau I don't quite understand this comment. From what I see in the system, when I create an extractor, it takes some time until a new entry is made in the table with the extractors. When the new entry is displayed, I can click on Review and go to the table where already suggestions are shown and I can start accepting/correcting.
This is what I describe here.

 Do I need to click "Find suggestions" before I start doing anything?

Reply:
> I think it is important to explain how the "Find suggestions" button work. 

> I also think it is important to give a basic notion on how to properly use this feature:
> 
> 1. Label some data, around 30-40 samples.
> 2. Run "Find suggestions"
> 3. Process some suggestions, particularly those that the ML model is not able to properly understand have more value than just confirming the patterns that the model is already doing well.
> 4. Back to step 2.

I tried to explain the "Find suggestions" function later in the "Retraining the system for improved suggestions" section. I will use your suggestion to make there the optimal flow more explicit.

Reply:
@txau  Second part is resolved with [ccb4fa8](https://github.com/huridocs/uwazi-documentation/pull/49/commits/ccb4fa86fc0a5fd46cbb0a52689dadbd9cde35cd)

Waiting feedback on the first part to fully resolve your comment

Reply:
I am merging this in the current status so that we have an online version for the beta testing and can iterate and improve on this based on more feedback